zerver/lib/test_classes.py

```diff
@@ -417,7 +417,7 @@ def submit_reg_form_for_user(
             from_confirmation: Optional[str]='', full_name: Optional[str]=None,
             timezone: Optional[str]='', realm_in_root_domain: Optional[str]=None,
             default_stream_groups: Optional[List[str]]=[],
-            source_realm: Optional[int]=0, **kwargs: Any) -> HttpResponse:
+            source_realm: Optional[str]='0', **kwargs: Any) -> HttpResponse:
```

Comment:
I could set source_realm to empty string value and check in registration.py whether source_realm is empty string to prevent conversion it to int but then I'll get next AssertionError. 

`======================================================================
FAIL: test_register (zerver.tests.test_signup.LoginTest)
----------------------------------------------------------------------
Traceback (most recent call last):
File "/home/eoanermine/zulip/zerver/tests/test_signup.py", line 607, in test_register
self.assertEqual(len(queries), 77)
AssertionError: 76 != 77

----------------------------------------------------------------------`

Reply:
Can someone explain to me that it checks and can I change it to

`python3
self.assertEqual(len(queries), 76)
`
All other tests ran okay.

Reply:
We should just use the default of None rather than `''` if we don't need to copy from a source realm, I think?

@hackerkid FYI can you confirm?